src/main/java/io/spine/protobuf/TypeConverter.java

```diff
@@ -56,6 +56,16 @@
  *         EnumValue} type and then is converted to the Java Enum through the value {@linkplain
  *         EnumValue#getName() name} or {@linkplain EnumValue#getNumber() number}.
  * </ul>
+ *
+ * <p>Note that the following types can only be converted in one direction,
+ * from {@linkplain Object Java Objects} to {@link Any}:
+ * <ul>
+ *     <li>{@link java.util.List Java List} types via {@link ListConverter}.
+ *     <li>{@link java.util.Map Java Map} types via {@link MapConverter}.
+ * </ul>
+ *
+ * <p>The converter throws {@link UnsupportedOperationException} if asked to perform
+ * an unsupported conversion.
```

Comment:
Let's talk specifically here. We already mentioned the "unsupported" thing in the name of the exception.